tests/test_server.py

```diff
@@ -44,6 +47,16 @@ def test_xeventlet(self):
         self.assertTrue(resp.ok)
         self.assertEqual(resp.content, b'<b>Hello xeventlet</b>!')
 
+    def test_simpl_server(self):
+        argv = ['server', '--quiet-server', '--port', str(get_free_port())]
+        proc = multiprocessing.Process(
+            target=simpl_cli.main, kwargs={'argv': argv})
```

Comment:
I struggled with this in https://github.com/checkmate/simpl/blob/master/tests/test_server.py#L109. I think this might not get correctly reflected in in test coverage.